cmake/compilers/GNU.cmake

```diff
@@ -75,8 +75,9 @@ set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -Wformat -Wformat-secur
 if (NOT APPLE AND NOT MINGW)
     set(TBB_LIB_LINK_FLAGS ${TBB_LIB_LINK_FLAGS} -Wl,-z,relro,-z,now,-z,noexecstack)
 endif()
-set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} $<$<NOT:$<CONFIG:Debug>>:-D_FORTIFY_SOURCE=2> )
-
+if (NOT CMAKE_CXX_FLAGS MATCHES "_FORTIFY_SOURCE")
+  set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} $<$<NOT:$<CONFIG:Debug>>:-D_FORTIFY_SOURCE=2> )
+endif ()
```

Comment:
Where was `_FORTIFY_SOURCE` defined first in your usage scenario? Do you have a control over this compile definition prior oneTBB compilation?

Reply:
this was set by the distribution CXXFLAGS environment variable

Reply:
I guess it makes sense to add such check to other compiler cmake files for consistency. That would be `Intel.cmake` and `Clang.cmake` as well.